File: C_programming/armstrongno.c
//29.Write a C program to check whether a given number is an armstrong number or not.
int main()
{
	int num,originalNum,remainder,result = 0;
	printf("Enter a three digit number :");
	scanf("%d",&num);
	originalNum=num;
	
	while(originalNum !=0)
	{
		//remainder contains the last digit
		remainder=originalNum % 10;
		
		result += remainder * remainder * remainder; 
		//removing last digit from the original no.
		originalNum /= 10;
		
	}
	if(result == num)
	printf("%d is an armstrong number.",num);
	else
	printf("%d is not  an armstrong number.",num);
	return 0;
}																							

File: Java_Programs/ArmstrongNo.java
//11.wap to accept a no. and then print a no. is armstrong no. or not.
import java.util.*;
class ArmstrongNo
{
public static void main(String args[])
{
	int num,originalnum,remainder,result=0;
	Scanner s=new Scanner(System.in);
	System.out.println("enter a number :");
	num=s.nextInt();
	originalnum=num;

	while(originalnum !=0)
	{
		remainder=originalnum % 10;
		result +=Math.pow(remainder,3);
		originalnum/=10;
	}
	if(result==num)
	System.out.println(num + " is an Armstrong number.");
	else
	System.out.println(num+ "is not an Armstrong number.");
 }
}




